easybuild/easyconfigs/o/OpenMM/OpenMM-7.1.1-intel-2017b-Python-3.6.3.eb

```diff
@@ -0,0 +1,50 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics 
+
+easyblock = "CMakeMake"
+
+name = "OpenMM"
+version = "7.1.1"
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://simtk.org/home/openmm'
+description = """OpenMM is a toolkit for molecular simulation."""
+
+toolchain = {'name': 'intel', 'version': '2017b'}
+
+source_urls = ['https://github.com/pandegroup/openmm/archive/']
+sources = ['%(version)s.tar.gz']
```

Comment:
@tovrstra please add an `sha256` ckecksum. You can use the option `--inject-checksums`.

Reply:
Done.